queries.sql

```diff
@@ -8,5 +8,52 @@ SELECT * FROM animals WHERE neutered=true;
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 SELECT * FROM animals WHERE name NOT IN('Gabumon');
 
+/*transaction queries*/
+BEGIN; 
+UPDATE animals SET species='unspecified';
+SELECT * as animals FROM animals;
+ROLLBACK;
 
```

Comment:
- [ ] As mentioned by the previous reviewer, he said `check if the changes were made before and after the rollback`, here you have only checked for the `before`, how about the `after` the rollback?